src/citrine/resources/gemtables.py

```diff
@@ -210,21 +209,17 @@ def initiate_build(self, config: Union[TableConfig, str, UUID], *,
             if version is None:
                 raise ValueError('Version must be specified when building by config uid.')
             uid = config
```

Comment:
```suggestion
            uid = config
        logger.info(f'Submitting table build for config {uid} version {version}...')
```
Removing non-existent job_id from logging statement.  Including log statement so client knows which request went into the ether.